cpp/src/sampling/detail/sampling_utils_impl.cuh

```diff
@@ -78,26 +78,63 @@ count_and_remove_duplicates(raft::handle_t const& handle,
     std::move(result_src), std::move(result_dst), std::move(result_wgt), std::move(result_count));
 }
 
-template <typename vertex_t, typename weight_t>
+template <typename vertex_t>
 struct return_all_edges_e_op {
```

Comment:
I think this is a misnomer. This does not return all edges but return an edge with all properties.